runtime/lua/vim/_editor.lua

```diff
@@ -1075,14 +1076,17 @@ function vim.deprecate(name, alternative, version, plugin, backtrace)
 
     if not is_hard_deprecated then
       return
+    elseif current_version >= removal_version then
+      will_be_removed = 'was removed'
     end
   end
 
   local msg = ('%s is deprecated'):format(name)
   msg = alternative and ('%s, use %s instead.'):format(msg, alternative) or (msg .. '.')
-  msg = ('%s%s\nThis feature will be removed in %s version %s'):format(
+  msg = ('%s%s\nFeature %s in %s %s'):format(
```

Comment:
Improve wording for case when `vim.deprecate()` is used to signal a change on the current `x.y-dev` version.